mmcv/runner/hooks/lr_updater.py

```diff
@@ -213,24 +219,27 @@ def get_lr(self, runner, base_lr):
 @HOOKS.register_module()
 class ExpLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, gamma, **kwargs):
+    def __init__(self, gamma: float, **kwargs) -> None:
         self.gamma = gamma
         super().__init__(**kwargs)
 
-    def get_lr(self, runner, base_lr):
+    def get_lr(self, runner: 'runner.BaseRunner', base_lr: float):
         progress = runner.epoch if self.by_epoch else runner.iter
         return base_lr * self.gamma**progress
 
 
 @HOOKS.register_module()
 class PolyLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, power=1., min_lr=0., **kwargs):
+    def __init__(self,
```

Comment:
We should unify the return value type hints of the `__init__` method.

Reply:
Maybe __init__ method don't need return ?

Reply:
Agree~